packages/ckeditor5-core/src/accessibility.ts

```diff
@@ -61,40 +62,52 @@ export default class Accessibility {
 			description: t( 'These keyboard shortcuts allow for quick access to content editing features.' )
 		} );
 
+		const navigationKeystrokes = [
+			{
+				label: t( 'Close contextual balloons, dropdowns, and dialogs' ),
+				keystroke: 'Esc'
+			},
+			{
+				label: t( 'Open the accessibility help dialog' ),
+				keystroke: 'Alt+0'
+			},
+			{
+				label: t( 'Move focus between form fields (inputs, buttons, etc.)' ),
+				keystroke: [ [ 'Tab' ], [ 'Shift+Tab' ] ]
+			},
+			{
+				label: t( 'Move focus to the toolbar, navigate between toolbars' ),
+				keystroke: 'Alt+F10',
+				mayRequireFn: true
+			},
+			{
+				label: t( 'Navigate through the toolbar or menu bar' ),
+				keystroke: [ [ 'arrowup' ], [ 'arrowright' ], [ 'arrowdown' ], [ 'arrowleft' ] ]
+			},
+			{
+				label: t( 'Execute the currently focused button' ),
+				keystroke: [ [ 'Enter' ], [ 'Space' ] ]
+			}
+		];
+
+		if ( isMenuBarVisible ) {
```

Comment:
I could keep the previous structure and add the inline logic, like:
```
	...(
			isMenuBarVisible ?
				[ {
					label: t( 'Move focus to the menu bar, navigate between menu bars' ),
					keystroke: 'Alt+F9',
					mayRequireFn: true
				} ] :
				[]
		),
```

However, the current approach seems more clear.